### DISCUSSION TITLE
Game crashing on startup with 1.17.1 fabric and some mods.

### DISCUSSION THREAD

**Discussion by Servoh** [FabricMC/fabric-api#1550] [Player support]
So, I am trying to play with fabric 1.17.1 and some mods, but I keep getting an error everytime.
The crash-log is [here](https://pastebin.com/WJY2vev5)

**Comment by mechanikate** [comment]
Did you update your mods folder? Try to update Carpet, perhaps.
You're using 1.17.1, and the Carpet version v1.4.40 is for 1.17
